### PR TITLE
Adding docs around the new "css" Encore functions and disabling file tracking

### PR DESCRIPTION
**Pull request by weaverryan**
Hi!

Docs for https://github.com/symfony/webpack-encore-bundle/pull/91

Cheers!

### CODE REVIEW

File: frontend.rst [hunk 1]
```diff
@@ -69,6 +69,7 @@ Guides
 
 * :doc:`Using Bootstrap CSS & JS </frontend/encore/bootstrap>`
 * :doc:`Creating Page-Specific CSS/JS </frontend/encore/page-specific-assets>`
+* :doc:`Rendering Multiple Templates in a Request: Emails, PDFs </frontend/encore/file-tracking>`
```
Comment (wouterj) [inline frontend.rst:72]: Why aren't we doing ``:doc:`/frontend/encore/file-tracking` `` here? :)

File: frontend/encore/file-tracking.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    <style>
+        {{ encore_entry_css_source('my_entry')|raw }}
+    </style>
+
+    <script>
+        {{ encore_entry_js_source('my_entry')|raw }}
+    </script>
+
+If you can't use these `encore_entry_*_source` functions, you can instead
```
Comment (OskarStark) [inline frontend/encore/file-tracking.rst:43]: Double backticks please

**Review by OskarStark — COMMENTED** [review]
Do we need a versionadded directive?

File: frontend/encore/file-tracking.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+manually disable and enable "file tracking":
+
+.. code-block:: html+twig
+
+    {% do encore_disable_file_tracking() %}
+        {{ encore_entry_link_tags('entry1') }}
+        {{ encore_entry_script_tags('entry1') }}
+    {% do encore_enable_file_tracking() %}
+
+With this, *all* JS and CSS files for `entry1` will be rendered and
```
Comment (wouterj) [inline frontend/encore/file-tracking.rst:53]:
```suggestion
With this, *all* JS and CSS files for ``entry1`` will be rendered and
```

File: frontend/encore/file-tracking.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Resetting the File Tracking
+---------------------------
+
+If using ``encore_disable_file_tracking()`` won't work for you for some
+reason, you can also "reset" EncoreBundle's internal cache so that the
+bundle re-renders CSS or JS files that it previously rendered. For
+example, in a controller::
+
+
```
Comment (wouterj) [inline frontend/encore/file-tracking.rst:64]:
```suggestion
```

File: frontend/encore/file-tracking.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    // src/Controller/SomeController.php
+
```
Comment (wouterj) [inline frontend/encore/file-tracking.rst:66]:
```suggestion
    namespace App\Controller;

```

File: frontend/encore/file-tracking.rst [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupInterface;
+
+    class SomeController
+    {
+        public function index(EntrypointLookupInterface $entrypointLookup)
+        {
+            $entrypointLookup->reset();
+            // render a template
```
Comment (wouterj) [inline frontend/encore/file-tracking.rst:74]:
```suggestion
            // ... render a template
```

File: frontend/encore/file-tracking.rst [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+            $entrypointLookup->reset();
+            // render another template
```
Comment (wouterj) [inline frontend/encore/file-tracking.rst:77]:
```suggestion
            // ... render another template
```

**Comment by javiereguiluz** [comment]
Let's close this old pending PR because we're moving away from Webpack Encore very fast so it doesn't make sense to try to add new complex features like this one. Thanks!

**Pull request closed without merging**